datamade/tutorials
# Refactor this repo and add stack change docs

## Overview

This PR makes three major changes:

- Refactors the repo to use a nested-directory structure for each area of expertise
- Adds documentation for our R&D and stack change process
- Adds preliminary adoption artifacts for GatsbyJS

## Testing instructions

- View the branch [on GitHub](https://github.com/datamade/tutorials/tree/jfc/stack-changes-and-gatsby) and confirm that the repo structure makes sense
- Read through new documentation and confirm it looks good
- Make sure that links resolve
